PyRoute/Galaxy.py

```diff
@@ -395,31 +410,34 @@ def write_routes(self, routes=None):
         if routes == 'xroute':
             path = os.path.join(self.output_path, 'stations.txt')
             with codecs.open(path, "wb", 'utf-8') as f:
-                stars = [star for star in self.stars if star.tradeCount > 0]
+                stars = [self.star_mapping[item] for item in self.stars]
+                stars = [star for star in stars if star.tradeCount > 0]
```

Comment:
```suggestion
                stars = [star for star in self.star_mapping.values() if star.tradeCount > 0]
```

Reply:
I space-cadetted _that_ a bunch of times, didn't I?